jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp

```diff
@@ -92,16 +92,16 @@ ecma_date_insert_num_with_sep (ecma_string_t **str_p, /**< input/output string *
 } /* ecma_date_insert_num_with_sep */
 
 /**
- * The Date.prototype object's 'toString' routine
+ * The Date.prototype object's 'toISOString' routine
  *
  * See also:
- *          ECMA-262 v5, 15.9.5.2
+ *          ECMA-262 v5, 15.9.5.43
```

Comment:
All builtin function names are already declared in `ecma-builtin-date-prototype.inc.h`. So we don't need to change the position of these functions.